### PR TITLE
Output what cuda archs rapids-cmake is building for

### PR DESCRIPTION
**Pull request by robertmaynard**
## Description
Fixes https://github.com/rapidsai/rapids-cmake/issues/608

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/rapids-cmake/blob/HEAD/CONTRIBUTING.md).

### CODE REVIEW

**Review by KyleFromNVIDIA — APPROVED** [review]
Approved with one minor suggestion, feel free to take it or leave it.

File: rapids-cmake/cuda/set_architectures.cmake [hunk 1]
```diff
@@ -80,4 +80,6 @@ function(rapids_cuda_set_architectures mode)
     set(CMAKE_CUDA_ARCHITECTURES ${CMAKE_CUDA_ARCHITECTURES} PARENT_SCOPE)
   endif()
 
+  message(STATUS "${CMAKE_PROJECT_NAME} CUDA architectures building for: ${CMAKE_CUDA_ARCHITECTURES}")
```
[the remark this answers is not in the document]
Comment (robertmaynard) [reply rapids-cmake/cuda/set_architectures.cmake:83]: That is a great idea. Accepted the changes

**Review by vyasr — APPROVED** [review]
Needs a style update, but otherwise LGTM

**Pull request merged**